tests/sentry/stacktraces/test_functions.py

```diff
@@ -103,6 +103,18 @@
         ],
         ["pthread_cond_timedwait@@GLIBC_2.3.2", "pthread_cond_timedwait"],
         ["glob64@GLIBC_2.2", "glob64"],
+        [
+            "static WinUI.WindowsXamlManager.getForCurrentThread() throws -> WinUI.WindowsXamlManager?",
```

Comment:
This example may be confusing to folks who don't expect to see Windows APIs in Swift code, but no biggie.

Reply:
Changed it to something more generic.